gradle/libs.versions.toml

```
@@ -17,6 +17,7 @@ mockito = "4.0.0"
 telemetryGenerator = "1.0.25"
 testLogger = "3.1.0"
 testRetry = "1.2.1"
+slf4j = "1.7.32"
```

They seem to be up to 1.7.36 in the interim - might as well rev to the latest while we're at it.
```suggestion
slf4j = "1.7.36"
```